### PR TITLE
IMP precision of using Requirement.txt

### PR DESCRIPTION
**Pull request by bealdav**
### Milestone of the branch
master

### Module(s)
MT

### Fixes / new features
UPD doc

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -989,10 +989,16 @@ Pull requests can be closed if:
...
 ### Repositories
 
+## Naming
+
 * Project name must not contain odoo or openerp
 * Project name for localization is "l10n_belgium" for Belgium
 * Project name for connectors is "connector-magento" for Magento connector
 
+## Branch configuration
+Python packages to install, must be preferably, define in requirement.txt than travis.yml file.
```
Comment (pedrobaeza) [inline CONTRIBUTING.md]: s/requirement/requirements

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -989,10 +989,16 @@ Pull requests can be closed if:
 
 ### Repositories
 
+## Naming
+
 * Project name must not contain odoo or openerp
 * Project name for localization is "l10n_belgium" for Belgium
 * Project name for connectors is "connector-magento" for Magento connector
 
+## Branch configuration
```
Comment (pedrobaeza) [inline CONTRIBUTING.md]: Include these sections in the index

Comment (bealdav) [inline CONTRIBUTING.md]: bad indent of my part 
#### Branch config

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -989,10 +989,16 @@ Pull requests can be closed if:
 
 ### Repositories
 
+#### Naming
+
 * Project name must not contain odoo or openerp
 * Project name for localization is "l10n_belgium" for Belgium
```
Comment (pedrobaeza) [inline CONTRIBUTING.md]: I'm seeing right now here a non related typo. It should be `l10n-belgium`. Can you change it?

**Comment by mourad-ehm** [comment]
:+1:

**Pull request merged**